templates/documents/single_collection_home.html

```diff
@@ -52,7 +58,9 @@ <h2 class="text-lg font-semibold">Files</h2>
               <div class="flex items-center gap-2">
                 {% if collection_file.status %}
                   <div class="tooltip" data-tip="{{ collection_file.status }}">
```

Comment:
the tooltip should use the status label from the enum instead of the value (`In Progress` instead of `in_progress`). My preferred way for doing that is to add a `<field_name>_enum` property to the model:

```python
@property
def status_enum(self):
    return FileStatus(self.status)
```

Reply:
Ah yes, thanks. https://github.com/dimagi/open-chat-studio/pull/1560/commits/70df13e715d5c89e83d31ad7cfabf23967c0fd33